src/services/review/ReviewRequestService.ts

```diff
@@ -393,9 +407,17 @@ export default class ReviewRequestService {
         const {
           title,
           body,
-          changed_files,
           created_at,
         } = await this.apiService.getPullRequest(siteName, pullRequestNumber)
+        const { files } = await this.apiService.getCommitDiff(siteName)
+        const filterPlaceholderFiles = files
+          .filter((file) => {
+            const extractPlaceholderFileResult = extractPathInfo(
+              file.filename
+            ).andThen((pathInfo) => this.extractPlaceholderInfo(pathInfo))
+            return extractPlaceholderFileResult.isErr()
+          })
```

Comment:
why not `extractPathInfo` -> `isPlaceholderFile`?

Reply:
makes sense